samples/cpp/cloning_demo.cpp

```diff
@@ -1,251 +0,0 @@
-/*
```

Comment:
Propose to add it as snippet for illuminationChange, colorChange, seamlessClone, textureFlattening